container-deployment/install-by-docker.md

```diff
@@ -88,30 +94,41 @@ $ docker logs 775a8e192f2b | tail -n 1
 ```
 
+
 ## Next Steps
 
-Now you know how to use Fluentd via Docker. Here're a couple of Docker
-related documentations for Fluentd.
+Now, you know how to use Fluentd via Docker.
+
+Here are some Docker related resources for Fluentd:
+
+- [Fluentd official Docker image](https://hub.docker.com/r/fluent/fluentd/)
+- [Fluentd official Docker image
+  (Source)](https://github.com/fluent/fluentd-docker-image)
+- [Docker Logging Driver and
+  Fluentd](/container-deployment/docker-logging-driver.md)
+- [Docker Logging via EFK (Elasticsearch + Fluentd + Kibana) Stack with Docker
+  Compose](/container-deployment/docker-compose.md)
 
--   [Fluentd's official Docker image](https://hub.docker.com/r/fluent/fluentd/)
--   [Fluentd's official Docker image (Source)](https://github.com/fluent/fluentd-docker-image)
--   [Docker Logging Driver and Fluentd](/container-deployment/docker-logging-driver.md)
--   [Docker Logging via EFK (Elasticsearch + Fluentd + Kibana) Stack with Docker Compose](/container-deployment/docker-compose.md)
+Also, refer to the following tutorials to learn how to collect data from various
+data sources:
 
-Also, please see the following tutorials to learn how to collect your
-data from various data sources.
+- Basic Configuration
+  - [Config File](/configuration/config-file.md)
+- Application Logs
+  - [Ruby](/language/ruby.md), [Java](/language/java.md),
+    [Python](/language/python.md), [PHP](/language/php.md),
+    [Perl](/language/perl.md), [Node.js](/language/nodejs.md),
+    [Scala](/language/scala.md)
+- Examples
+  - [Store Apache Log into Amazon S3](/guides/apache-to-s3.md)
+  - [Store Apache Log into MongoDB](/guides/apache-to-mongodb.md)
+  - [Data Collection into HDFS](/guides/http-to-hdfs.md)
 
--   Basic Configuration
-    -   [Config File](/configuration/config-file.md)
--   Application Logs
-    -   [Ruby](/language/ruby.md), [Java](/language/java.md), [Python](/language/python.md), [PHP](/language/php.md),
-        [Perl](/language/perl.md), [Node.js](/language/nodejs.md), [Scala](/language/scala.md)
--   Examples
-    -   [Store Apache Log into Amazon S3](/guides/apache-to-s3.md)
-    -   [Store Apache Log into MongoDB](/guides/apache-to-mongodb.md)
-    -   [Data Collection into HDFS](/guides/http-to-hdfs.md)
 
 ------------------------------------------------------------------------
 
-If this article is incorrect or outdated, or omits critical information, please [let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open).
-[Fluentd](http://www.fluentd.org/) is a open source project under [Cloud Native Computing Foundation (CNCF)](https://cncf.io/). All components are available under the Apache 2 License.
+If this article is incorrect or outdated, or omits critical information, please
+[let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open).
+[Fluentd](http://www.fluentd.org/) is an open-source project under [Cloud Native
```

Comment:
@repeatedly: Why is this working? It is being rendered okay.
Is it different from the rest of the page? Special case?